src/components/ConversationSettings/LobbySettings.vue

```diff
@@ -54,6 +54,7 @@
 							id="moderation_settings_lobby_timer_field"
 							aria-describedby="moderation_settings_lobby_timer_hint"
 							:value="lobbyTimer"
+							:defaultValue="defaultLobbyTimer"
```

Comment:
Attribute ':defaultValue' must be hyphenated

```suggestion
							:default-value="defaultLobbyTimer"
```